search/search_params.hpp

```diff
@@ -49,7 +49,7 @@ class SearchParams
 
   double m_lat, m_lon;
 
-  // A minimum distance between search results in mercator, needed for
+  // A minimum distance between search results in meters, needed for
   // pre-ranking of viewport search results.
   double m_minDistanceOnMapBetweenResults;
```

Comment:
[optional] This struct looks a bit irregularly - some members are accessible via set/get pairs, while other are accessible directly. May be do it in a one manner? It is not about the PR, just as concomitant note.